packages/core/src/material/PBRMaterial.ts

```diff
@@ -222,6 +228,66 @@ export class PBRMaterial extends PBRBaseMaterial {
     }
   }
 
+  /**
+   * Sheen color.
+   * @defaultValue `[0,0,0]`
+   */
+  get sheenColor(): Color {
+    return this.shaderData.getColor(PBRMaterial._sheenColorProp);
+  }
+
+  set sheenColor(value: Color) {
+    const sheenColor = this.shaderData.getColor(PBRMaterial._sheenColorProp);
+    if (value !== sheenColor) {
+      sheenColor.copyFrom(value);
+    }
+  }
+
+  /**
+   * Sheen roughness, from 0.0 to 1.0.
+   * @defaultValue `0.0`
+   */
+  get sheenRoughness(): number {
+    return this.shaderData.getFloat(PBRMaterial._sheenRoughnessProp);
+  }
```

Comment:
Maybe should add ` value = Math.max(0, Math.min(1, value));` if you want clamp the range like comments